util-core/src/test/scala/com/twitter/concurrent/AsyncStreamTest.scala

```diff
@@ -322,14 +322,14 @@ class AsyncStreamTest extends FunSuite with GeneratorDrivenPropertyChecks {
   }
 
   test("takeWhile") {
-    forAll { (as: List[Int], p: Int => Boolean) =>
```

Comment:
for folks who are curious about why this didn't work before, scalacheck generated functions were constant.  we should be able to use scalacheck generated functions if we upgrade to 0.13.0.

https://github.com/rickynils/scalacheck/pull/171